crates/hyperdrive-math/src/lib.rs

```diff
@@ -128,18 +134,23 @@ impl State {
         time - time % self.config.checkpoint_duration
     }
 
-    pub fn time_remaining_scaled(
+    /// Gets the normalized time remaining
+    fn calculate_normalized_time_remaining(
```

Comment:
This will need to be public for you to be able to use it.

Reply:
I don't think we want this as part of the public API. This function is only intended to be called internally by other rust sdk methods.

Reply:
Yeah makes sense. I left that comment before getting deeper into the PR